docs/howto/auth/ldap.rst

```diff
@@ -0,0 +1,46 @@
+.. _howto/auth/ldap:
```

Comment:
I think we should either make this be fully complete and talk about at least the required parameters of LDAPAuthenticator, or remove it.

Reply:
the main reason I added it there was because I was confused that LDAP was getting "special treatment" on the main authenticator page. We could just add contextualizing language there saying "we're just calling out LDAP for example" or something. WDYT?